app/assets/javascripts/plate-map/plate-layout.js

```diff
@@ -167,5 +173,112 @@ $.widget("DNA.plateLayOut", {
   // wellsData follows syntax: {0:{field1: val1, field2: val2}, 1:{field1: val1, field2: val2}}
   getWellsDifferences: function(wellsData) {
     return this.getDifferentWellsVals(wellsData);
+  },
+
+  disableAllFields: function(){
+    var numericFields = $('.plate-setup-tab-input, .plate-setup-tab-label-select-field, ' +
+      '.plate-setup-tab-multiselect-field, .plate-setup-tab-select-field');
+    for (var i = 0; i < numericFields.length; i++) {
+      numericFields[i].disabled = true;
+    }
+    $(".plate-setup-tab-name-singleSelect").text("Select to inspect");
+  },
+
+  enableAllFields: function(){
```

Comment:
disableAllFields/enableAllFields should be `setFieldsEnabled(true/false)`

Reply:
refactored